README.md

```diff
@@ -1,163 +1,263 @@
-# Lexical scoping
+# Lexical Scope
 
 ## Learning Goals
 
-- Explain the concept of lexical scoping.
-- Describe how lexical scoping informs the scope chain of a function.
+- Describe lexical scope as the scope where a variable or function is defined.
+- Examine how lexical scoping informs the scope chain of a function.
 
 ## Introduction
 
-In this lesson, we'll learn how JavaScript decides which outer scope to place
-into the scope chain for a new function.
+In our journey to understand scope, we've learned about the three different
+levels, the scope chain, and how JavaScript traverses it. What we haven't
+learned is how JavaScript decides which outer scope to place into the scope
+chain for a new function.
 
-First, a brief note. Some of the material introduced in this section might feel
-a bit confusing or esoteric, but, fear not, you're going to get more and more
-comfortable with these concepts throughout this course. If you're finding it
-difficult to wrap your brain around some of these more abstract concepts at this
-point, **don't freak out**. This is really difficult material that even seasoned
-programmers occasionally struggle with. After you've read the material and given
-it a college try, feel free to move on to the next lesson — with a mental note
-to return once you have more JavaScript experience under your belt. Okay,
-onwards and upwards!
+We have an _idea_ of how, based on what we learned about the chain and nesting
+scopes. Anything that a scope is nested within would be considered part of its
+outer scope chain.
 
-Take a look at the following code snippet:
+However, that doesn't give the whole picture. It doesn't get to the root core of
+how JavaScript decides where scope is defined when a variable, function, or code
+block is written. This is when the concept of lexical scope comes in.
 
-```js
-const myVar = "Foo";
+## Lexical Scope
 
-function first() {
-  console.log("Inside first()");
+So, what _is_ this magical lexical scope? Before we break it down, let's look at
+the following code snippet:
 
-  console.log("myVar is currently equal to:", myVar);
-}
+```js
+const currentStatus = "Crocheting a sweater 🧶";
 
-function second() {
-  const myVar = "Bar";
+function showStatus() {
+  console.log("Current Status:");
+  console.log(" ", currentStatus);
+}
 
-  first();
+function changeStatus() {
+  const currentStatus = "Cat tangled all my yarn! 🙀";
+  showStatus();
 }
 ```
 
-Think about what we've learned in previous lessons about how JavaScript looks up
-the scope chain to perform identifier resolution. Given that information, what
-do you think will get logged out to the console when we invoke `second()`? Let's
-try it out:
+What do you think will be logged to the console when we run `changeStatus()`?
```

Comment:
This is a great example of built in _Check for Understanding_ questions. It doesn't require the instructor to monitor or grade anything, but the students get to monitor their own understanding along the way.